### ISSUE TITLE
Can't Install With githubUrl in Unity 

### ISSUE THREAD

**Issue by lemonip** [Cysharp/MemoryPack#290]
1. I used unity ver 2023.2.1f1
2. Install package from this Git Url
https://github.com/Cysharp/MemoryPack.git?path=src/MemoryPack.Unity/Assets/MemoryPack.Unity
![t1](https://github.com/Cysharp/MemoryPack/assets/76773354/f280b5d8-98a7-4b4a-962e-5d9673b50ba4)

3. Get error message
![t2](https://github.com/Cysharp/MemoryPack/assets/76773354/4b3929be-a841-47a4-9b7d-d4d159432eb9)

4. Get Unsafe And Complie
![t3](https://github.com/Cysharp/MemoryPack/assets/76773354/75a78742-1c45-4ecb-88a1-5bbb1c3f3566)

5. Still Get Same Error Message.

**Comment by neuecc** [comment]
Are you installed Core library from NuGet?

**Comment by lemonip** [comment]
I'm using visual studio.
If you're asking me to install the nugget package on the IDE, I've never
done it.
Is it correct to ask for these installations?
Install NuGet Packages in Visual Studio, VS Code, and Rider | General
Information | DevExpress Documentation
<https://docs.devexpress.com/GeneralInformation/116698/nuget/setup-visual-studios-nuget-package-manager>

2024년 5월 8일 (수) 오후 6:33, Yoshifumi Kawai ***@***.***>님이 작성:

**Comment by lemonip** [comment]
GlitchEnzo/NuGetForUnity: A NuGet Package Manager for Unity (github.com)
<https://github.com/GlitchEnzo/NuGetForUnity>
I also installed the memory pack core in nugget in Unity, but it didn't
work.

2024년 5월 9일 (목) 오후 11:26, ip lemon ***@***.***>님이 작성:

**Comment by lemonip** [comment]
When installed with nugget, these error messages are output to the console.

Assembly
'Assets/Packages/MemoryPack.Generator.1.21.1/analyzers/dotnet/cs/MemoryPack.Generator.dll'
will not be loaded due to errors:
Unable to resolve reference 'Microsoft.CodeAnalysis'. Is the assembly
missing or incompatible with the current platform?
Reference validation can be disabled in the Plugin Inspector.
Unable to resolve reference 'Microsoft.CodeAnalysis.CSharp'. Is the
assembly missing or incompatible with the current platform?
Reference validation can be disabled in the Plugin Inspector.

So I tried to install it with git, and the result is what I inquired about.
Different error messages are output.

2024년 5월 9일 (목) 오후 11:28, ip lemon ***@***.***>님이 작성:
